mount_linux.go

```diff
@@ -84,6 +91,10 @@ func directmount(dir string, cfg *MountConfig) (*os.File, error) {
 	}
 	delete(opts, "subtype")
 	data += "," + mapToOptionsString(opts)
+
+	if cfg.DebugLogger != nil {
+		cfg.DebugLogger.Println("Starting the unix mounting")
+	}
```

Comment:
should we add a log after unix.mount to mention that it returned. Most of the times we are seeing mounting started message but not completed message. This will help know if the mounting command is stuck.